src/Catel.MVVM/Catel.MVVM.Shared/MVVM/Commands/Command.cs

```diff
@@ -244,7 +244,7 @@ public void Execute(object parameter)
         public void Execute(TExecuteParameter parameter)
         {
 #pragma warning disable 4014
-            ExecuteAsync(parameter, false);
+            ExecuteAsync(parameter, false).Wait();
```

Comment:
Not sure if .Wait() is the right. I thought only the TaskCommand had async execution. Need to look into this.